projects/canopy/src/lib/forms/input/input.notes.ts

```diff
@@ -93,6 +93,7 @@ Prefix and suffix text can be added to input fields using the \`lgSuffix\` and \
 |------|-------------|:----:|:-----:|:-----:|
 | \`\`id\`\` | HTML ID attribute, auto generated if not provided. This will also propagate to the input 'id' and form 'for' attribute | string | 'lg-input-\${nextUniqueId++}' | No |
 | \`\`block\`\` | Property to make the input full width (for small screens only). | boolean | false | no
+| \`\`showLabel\`\` | Shows or visually hide the label | boolean | true | No |
```

Comment:
Really minor, but I think this should be grammatically more correct.
```suggestion
| \`\`showLabel\`\` | Show or visually hide the label | boolean | true | No |
```

Reply:
Done 👍